src/main/scala/cromwell/engine/backend/local/SharedFileSystem.scala

```diff
@@ -29,14 +35,19 @@ object SharedFileSystem {
     case unsupported => throw new UnsupportedOperationException(s"Localization strategy $unsupported is not recognized")
   })
 
+  val DockerLocalizers = localizePathAlreadyLocalized _ +: (LocalizationStrategies collect {
+    case "hard-link" => localizePathViaHardLink _
+    case "copy" => localizePathViaCopy _
+  })
```

Comment:
Yes, but especially because `Localizers` will throw an error if there are unrecognized localizer I thought there was no reason to throw it again here. 
Since they use the same configuration as input, if there is something unrecognized it will be detected by `Localizers` anyway.